spec/govuk_design_system_formbuilder/builder/error_summary_spec.rb

```diff
@@ -472,6 +472,15 @@ def formatted_error_messages
             end
           end
         end
+
+        context "when the custom presenter is an class that doesn't implement #formatted_error_messages" do
+          let(:non_presenter) { OpenStruct }
+          subject { builder.send(*args, presenter: non_presenter) }
+
+          specify "fails with an appropriate error message" do
+            expect { subject }.to raise_error(ArgumentError, "error summary presenter doesn't implement #formatted_error_messages")
+          end
+        end
       end
 
       context "as an instance" do
```

Comment:
nit - not related so ignore as you like but - rubocop-rspec perfers `with/without/when`
```suggestion
      context "with an instance" do
```

Reply:
yeah - i disagree with Rubocop in this instance (and several others!) 😄